pages/_wordid.vue

```diff
@@ -41,14 +41,20 @@ export default defineComponent({
     let description;
 
     if (i18n.locale === "en") {
-      title = `"${word.en}" is "${word.zhCN}" in Chinese`;
-      description = `"${word.en}" is "${word.zhCN}" in Chinese and "${word.ja}" in Japanese. This website contains English, Chinese, and Japanese translations for terms in Genshin Impact.`;
+      title = `"${word.en}" is ` + (word.zhCN ? `"${word.zhCN}" in Chinese` : `${word.ja} in Japanese`);
+      description = `"${word.en}" is `
+                  + (word.zhCN ? `"${word.zhCN}" in Chinese and ` : "")
+                  + `"${word.ja}" in Japanese. This website contains English, Chinese, and Japanese translations for terms in Genshin Impact.`;
     } else if (i18n.locale === "ja") {
       title = `「${word.ja}」は英語で "${word.en}"`;
-      description = `「${word.ja}」の英語表記は "${word.en}"、中国語表記は「${word.zhCN}」 ― このサイトはゲーム「原神」の用語の、日本語・英語・中国語の対訳を掲載しています。`;
+      description = `「${word.ja}」の英語表記は "${word.en}"`
+                  + (word.zhCN ? `、中国語表記は「${word.zhCN}」` : "")
+                  + " ― このサイトはゲーム「原神」の用語の、日本語・英語・中国語の対訳を掲載しています。";
     } else if (i18n.locale === "zh-CN") {
-      title = `"${word.zhCN}"的英语和日语翻译`;
-      description = `"${word.zhCN}"的英语是"${word.en}"，日语是"${word.ja}"。`; // TODO TranslationChanged
+      title = `"${word.zhCN ?? word.en}"的英语和日语翻译`;
+      description = word.zhCN ?
+        `"${word.zhCN}"的英语是"${word.en}"，日语是"${word.ja}"。` : // TODO TranslationChanged
+        `"${word.en}"的日语是"${word.ja}"。`;
```

Comment:
@Bill-Haku
I'm changing the title and the description of the word pages (e.g. https://genshin-dictionary.com/zh-CN/tulaytullahs-remembrance/) because if the word does not have Chinese translation yet, the titles and the descriptions would be weird like `""的英语和日语翻译`. (See the description of this PR for the titles and descriptions in all languages)

I changed the Chinese title and description like follows when Chinese translation is not available:

title: "ENGLISH"的英语和日语翻译
description: "ENGLISH"的日语是"JAPANESE"。

Do you think those are OK?
If you have a better idea of the Chinese title and description for the words without Chinese translation, I'm happy to change them.

Reply:
The description part is great, but in the title, it would be better to be written in `"ENGLISH"的日语翻译` if the Chinese translation is missing. In a specific word:
```
title = word.zhCN ? `"${word.zhCN}"的英语和日语翻译` : `"${word.en}"的日语翻译`
```

Reply:
Sounds nice! Thanks!